wallet/src/main/java/com/android/identity_credential/wallet/PresentationActivity.kt

```diff
@@ -113,9 +109,67 @@ class PresentationActivity : ComponentActivity() {
         RESPONSE_SENT,
     }
 
+    // reference WalletApplication for obtaining dependencies
+    private val walletApp: WalletApplication by lazy {
+        application as WalletApplication
+    }
+
     private var deviceRequest: ByteArray? = null
     private var deviceRetrievalHelper: DeviceRetrievalHelper? = null
-    private lateinit var sharedPreferences: SharedPreferences
+
+    // Transfer helper facilitates starting processing a presentation request and obtaining the
+    // response bytes once processing has finished. This enables showing one or more dialogs to
+    // the user to accept before sending a response to requesting party.
+    private var transferHelper: TransferHelper? = null
+
+    // get trust manager from WalletApplication
```

Comment:
this comment seems unnecessary, given the simplicity of the code below - I see maybe 3-4 more similar instances in this part of the class and imo its less readable as a result, would you mind removing some of the comments that just restate the code?